react-components/src/util/dialog/DEAlertDialog.js

```diff
@@ -0,0 +1,49 @@
+/**
+ *
+ * @author sriram
+ *
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { injectIntl } from "react-intl";
+import withI18N, { getMessage } from "../I18NWrapper";
+import intlData from "./messages";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import Dialog from "@material-ui/core/Dialog";
+import DEDialogHeader from "./DEDialogHeader";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogActions from "@material-ui/core/DialogActions";
+import Button from "@material-ui/core/Button";
+
+DEAlertDialog.propTypes = {
+    dialogOpen: PropTypes.bool.isRequired,
+    heading: PropTypes.string.isRequired,
+    alertMessage: PropTypes.string.isRequired,
+    handleClose: PropTypes.func.isRequired,
+};
+
+function DEAlertDialog(props) {
+    const [dialogOpen, heading, alertMessage, handleClose] = props;
+    return (
+        <Dialog
+            open={dialogOpen}
+            onClose={handleClose}
+            aria-describedby="alert-dialog-description"
+        >
+            <DEDialogHeader heading={heading} onClose={handleClose} />
+            <DialogContent>
+                <DialogContentText id="alert-dialog-description">
```

Comment:
Is this id meant to be a static ID?  If so, I would imagine we could have multiple alert dialogs open, in which case, this ID wouldn't be unique and could cause issues.

Reply:
There will be one alert dialog at a time because this is Modal but I agree, that `id` needs to be more unique. I will add a `parent-debug-id` prop from which a unique id can built.